### PR TITLE
virsh_define: Fix AttributeError

### PR DESCRIPTION
**Pull request by liuyd96**
Some tests need to set invalid attribute "foot" to video. If guest xml
doesn't have video device test will hit AttributeError.

Check guest xml and add video device if needed.

Signed-off-by: Liu Yiding <liuyd.fnst@fujitsu.com>

Before
```
[root@hpe-apollo80-01-n00 ~]# avocado run --vt-type libvirt --vt-arch aarch64 --vt-machine-type arm64-mmio virsh.define.positive_test.option_none
JOB ID     : fbb474ebdee15aad7394be87055d85ca354a3223
JOB LOG    : /root/avocado/job-results/job-2021-08-20T03.02-fbb474e/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.define.positive_test.option_none: ERROR: 'NoneType' object has no attribute 'set' (7.39 s)
RESULTS    : PASS 0 | ERROR 1 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 8.15 s
```

After
```
[root@hpe-apollo80-01-n00 ~]# avocado run --vt-type libvirt --vt-arch aarch64 --vt-machine-type arm64-mmio virsh.define.positive_test.option_none
JOB ID     : 406e0dc4ff5aa73512025af96dc7e40405cb6eb7
JOB LOG    : /root/avocado/job-results/job-2021-08-20T03.00-406e0dc/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.define.positive_test.option_none: PASS (7.76 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 8.53 s
```

### CODE REVIEW

**Comment by liuyd96** [comment]
@chloerh Please help review it. Thanks.

File: libvirt/tests/src/virsh_cmd/domain/virsh_define.py [hunk 1]
```diff
@@ -43,7 +44,19 @@ def prepare_xml_file():
...
+
+        # Add video device if guest doesn't have
+        if not new_xml.get_devices(device_type="video"):
+            logging.debug("Guest doesn't have video device, add one")
+            new_video = Video()
+            new_video.model_type = "virtio"
+            new_xml.add_device(new_video)
+            logging.debug("The xml with video is %s", new_xml)
+
+        # Set invalid attribute for video device
```
Comment (chunfuwen) [inline libvirt/tests/src/virsh_cmd/domain/virsh_define.py:56]: perhaps you may call new_xml.sync() before call new_xml.xmltreefile.find('/devices/video/model')

**Review by chunfuwen — APPROVED** [review]
LGTM

**Review by chloerh — APPROVED** [review]

**Comment by chloerh** [comment]
@Yingshun Could you help review this too? Thanks.

**Review by Yingshun — APPROVED** [review]
LGTM

**Comment by chloerh** [comment]
Thanks @Yingshun . I'll merge this.

**Pull request merged**